packages/apputils/src/toolbar.tsx

```diff
@@ -2,7 +2,7 @@
 // Distributed under the terms of the Modified BSD License.
 
 import {
-  IIterator, find, map
+  find, IIterator, map
```

Comment:
is this a prettier-ism which ignores ASCII case order?

Reply:
I don't think prettier was run on this and I don't think that prettier modifies the AST at all, so it won't reorder anything, just do wrapping, enforce, semi-colons no semi-colons, single quotes vs. double quotes, etc.

Reply:
Just curious why the import order was swapped.